pkg/component/etcd/etcd/crd.go

```diff
@@ -20,18 +21,20 @@ import (
 	"github.com/gardener/gardener/pkg/component"
 	"github.com/gardener/gardener/pkg/utils/flow"
 	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
+	kubernetesutils "github.com/gardener/gardener/pkg/utils/kubernetes"
 )
 
 var (
 	//go:embed crds/templates/crd-druid.gardener.cloud_etcds.yaml
-	// CRD holds the etcd custom resource definition template
-	CRD string
+	// CrdCloudEtcds holds the etcd custom resource definition template
+	CrdCloudEtcds string
```

Comment:
Why is this changed and what is a CloudEtcd? 😃 

Please revert.

Reply:
I aligned the variable name to the other packages, aas the naming follows a pattern of the filename, in this case `cloud_etcds.yaml`, therefore I aligned it.
Can also revert it back, if needed.